src/modules/_networking/local_network_gateways/local_network_gateway.tf

```diff
@@ -0,0 +1,7 @@
+resource "azurerm_local_network_gateway" "main" {
+  name                = var.settings.name
+  location            = local.location
+  resource_group_name = local.resource_group_name
+  gateway_address     = var.settings.gateway_address
+  address_space       = var.settings.address_space
```

Comment:
To ensure consistency, this should be `cidr`. Make sure to update examples as well